### DISCUSSION TITLE
Promise.all for persistAndFlush operations

### DISCUSSION THREAD

**Discussion by sergeyampo** [mikro-orm/mikro-orm#2788] [Q&A] [answered]
Why does Promise.all for em.persistAndFlush operation not work if I try to save multiple entities?
`Promise.all(em.persistAndFlush(book), em.persistAndFlush(paragraphs));`

**Comment by B4nan** [comment, marked as answer]
It should work in v5. As you havent provided any details, I dont have better answer.

**Reply by sergeyampo** [reply]
Book -> Paragraph. 1 -> M relation.
v. 4.5.10. Node 16.12
Paragraphs is an array of em.create(Paragraph) objects.
If I execute that Promise.all only books will be inserted.

**Reply by B4nan** [reply]
This works only in v5.
